src/main.rs

```diff
@@ -16,19 +16,77 @@ mod snapshot_tests;
 fn main() {
     human_panic::setup_panic!();
 
-    let Cargo::SemverChecks(args) = Cargo::parse();
+    let Cargo::SemverChecks(mut args) = Cargo::parse();
+
+    let feature_flags = HashSet::from_iter(args.unstable_features);
 
     configure_color(args.color_choice);
     let mut config = GlobalConfig::new();
+    config.set_log_level(args.check_release.verbosity.log_level());
+    config.set_feature_flags(feature_flags);
+
+    validate_feature_flags(&mut config, &mut args.unstable_options);
+
+    if config.feature_flag_enabled(FeatureFlag::HELP) {
+        config
+            .log_info(|config| {
+                let header = Style::new()
+                    .bold()
+                    .fg_color(Some(Color::Ansi(AnsiColor::Cyan)));
+                let option = Style::new().bold();
+
+                writeln!(config.stderr(), "{header}Unstable options{header:#}\n")?;
```

Comment:
Whoa fascinating, what's `{header:#}` here? Never seen that before, is that a fancy way to undo the style by using the alternate representation?

Reply:
Yeah, I was looking through `cargo` code and they use this way of using (semantically) named styles, which let you use the alternate formatting flag to render a reset code.

Reply:
Very cool, TIL!